README.md

```diff
@@ -46,6 +90,18 @@ helm install --name my-release fn
 
 > Note: if you do not pass the --name flag, a release name will be auto-generated. You can view releases by running helm list (or helm ls, for short).
 
+## Working with Fn 
+
+#### Ingress controller
+
+Please ensure that your ingress controller is running and has public-facing IP address.
+All you need to know that an ingress controller acts as a proxy between internal and public networks.
```

Comment:
An ingress controller acts as a proxy between your internal and public networks.